examples/platforms/nrf528xx/CMakeLists.txt

```diff
@@ -0,0 +1,83 @@
+#
+#  Copyright (c) 2020, The OpenThread Authors.
+#  All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions are met:
+#  1. Redistributions of source code must retain the above copyright
+#     notice, this list of conditions and the following disclaimer.
+#  2. Redistributions in binary form must reproduce the above copyright
+#     notice, this list of conditions and the following disclaimer in the
+#     documentation and/or other materials provided with the distribution.
+#  3. Neither the name of the copyright holder nor the
+#     names of its contributors may be used to endorse or promote products
+#     derived from this software without specific prior written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+#  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+#  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+#  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+#  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+#  POSSIBILITY OF SUCH DAMAGE.
+#
+
+if(OT_NCP_SPI)
+    target_compile_definitions(ot-config INTERFACE "SPIS_AS_SERIAL_TRANSPORT=1")
+else()
+    option(OT_USB "enable nrf USB as serial transport support")
+    if(OT_USB)
+        target_compile_definitions(ot-config INTERFACE "USB_CDC_AS_SERIAL_TRANSPORT=1")
+    else()
+        target_compile_definitions(ot-config INTERFACE "UART_AS_SERIAL_TRANSPORT=1")
+    endif()
+endif()
+
+set(NRF_COMM_SOURCES
+    src/alarm.c
+    src/diag.c
+    src/entropy.c
+    src/fem.c
+    src/flash.c
+    src/logging.c
+    src/misc.c
+    src/radio.c
+    src/system.c
+    src/temp.c
+)
+
+set(NRF_TRANSPORT_SOURCES
+    src/transport/spi-slave.c
+    src/transport/transport.c
+    src/transport/uart.c
+    src/transport/usb-cdc-uart.c
+)
+
+set(NRF_SINGLEPHY_SOURCES
+    src/flash_nosd.c
+)
+
+set(NRF_SOFTDEVICE_SOURCES
+    src/flash_sd.c
+    src/softdevice.c
+)
+
+set(NRF_INCLUDES
+    ${CMAKE_CURRENT_SOURCE_DIR}/src
+    ${PROJECT_SOURCE_DIR}/examples/platforms
+    ${PROJECT_SOURCE_DIR}/src/core
+)
+
+if(OT_PLATFORM STREQUAL "nrf52840")
+    set(NRF52840_3RD_LIBS
+        nordicsemi-nrf52840-radio-driver
+        nordicsemi-nrf52840-radio-driver-softdevice
+        nordicsemi-nrf52840-sdk
+        jlinkrtt
+    )
+    include(nrf52840/nrf52840.cmake)
```

Comment:
where is nrf52811.cmake?

Reply:
@bukepo  this pr was created for nrf52840,  should we separate nrf52811, nrf52833 or just add them in this PR?